playground/infrastructure/test_cd_helper.py

```diff
@@ -1,165 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Please don't delete the file. Instead leave at least empty with the comment "fixed in PR ... by adding Datastore client"